RoutingStrategies.xaml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--!
+    Name: Carissa Engebose, Pachia Lee, and Hannah Hotchkiss
+    Date: 10/16/24
+    Description: This implements and displays the routing strategies screen for the airports lab.
+                 It displays the starting airports, max distance, number of airports visited and 
+                 suggested routes as well as a map.
+    Bugs: The back button that was on the screen isn't there right now because navigation 
+          to this page has to come from planning tools. Also, we couldn't get the map to display 
+          a location, but the map is there.
+    Reflection: Overall this lab was very useful to learn about tabs and how to get a map for an app. 
+                There were a lot of difficulties especially with the map not working how we wanted it to, 
+                but it was nice to get practice on tabbed pages and work with new people.
+-->
+<TabbedPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:local="clr-namespace:Lab6_Starter"
+            xmlns:maps="clr-namespace:Microsoft.Maui.Controls.Maps;assembly=Microsoft.Maui.Controls.Maps"
+            x:Class="Lab6_Starter.RoutingStrategies"
+            Title="Routing Strategies"
+            xmlns:android="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
+            android:TabbedPage.ToolbarPlacement="Bottom">
+
+    <ContentPage Title="Airports"
```

Comment:
Once you've extracted this from the TabbedPage, you should change the title here to Routing Strategies.